packages/truffle-db/src/artifacts/schema.ts

```diff
@@ -75,5 +96,61 @@ export const schema = mergeSchemas({
       },
       
     },
+
+    Contract: {
```

Comment:
@gnidan below lies the heart of my problems right now. I tried a few different things here as far as fragments, etc. and I can't seem to get ast (actually now its shows up as {} but I believe it should be populated) or source data for the constructor).

Reply:
Should be okay now I think